setup/baseConfig.yaml

```diff
@@ -179,7 +179,7 @@ s3storage:
   bucket-name: <name of the bucket to be mounted. Default - first accessible bucket>
   key-id: <S3 access key ID. Default - use credentials from environment variables or shared profile (see README)>
   secret-key: <S3 secret access key. Default - use credentials from environment variables or shared profile (see README)>
-  region: <S3 region. Default - us-east-1>
+  region: <S3 region. If endpoint is provided, will read region from endpoint. Default - us-east-1>
```

Comment:
I tried to rephrase this a bit better. The basic idea is that if the user does not specify a region, then if the user provides an endpoint it uses the region in that. Otherwise, it uses the default.

Reply:
I think this is clearer. Thanks!